cla-frontend-project-console/src/ionic/constants/general.ts

```diff
@@ -11,6 +11,6 @@ export const generalConstants = {
     createTicketURL: 'https://jira.linuxfoundation.org/servicedesk/customer/portal/4',
     easyCLAHelpURL: 'https://lf-docs-linux-foundation.gitbook.io/easycla/getting-started/easycla-faqs',
     getAccessURL: 'https://jira.linuxfoundation.org/servicedesk/customer/portal/4',
-    CLA_GROUP_NAME_REGEX: /^([\w\p{L}][\w\s\p{L}()\[\]\.\,+-_]*)$/,
+    ALLOW_ALL_LANGUAGES: /^[^*|\":<>[\]{}`\\#%!~';@&$]+$/,
```

Comment:
What is this regex? What does it allow?  I don't see it allowing A-Zaz-z0-9. Are you able to write a unit test for this?

Reply:
Instead of allowing characters, I made regex of restricted characters so I will support all the languages frontend.